wasmCloud/examples
# Added wasmcloud.toml for each actor

## Feature or Problem
This PR is part 1 out of a 3 step process to improve our example documentation, which can be seen below

- [x] Add `wasmcloud.toml` files for all actor examples, so that each one just requires that someone types `wash build` to build and sign them
- [x] Add `wadm.yaml` files for all actor examples, so each example can be deployed simply with any prerequisites, and then `wash app put wadm.yaml; wash app deploy <example>`
- [ ] Rework example README's to use `wash build` and `wash app` for deploying the sample

## Related Issues
https://github.com/wasmCloud/project-templates/pull/65
https://github.com/wasmCloud/wasmcloud.com-dev/pull/71

## Release Information
N/A

## Consumer Impact
The goal of this procedure is to make examples more digestible and easy to run

## Testing



Built on platform(s)


Tested on platform(s)

### Unit Test(s)


### Acceptance or Integration


### Manual Verification
I ran `wash build` to make sure friendly names, capabilities, and call aliases carried over
